crates/interpreter/src/exec.rs

```diff
@@ -1058,25 +1064,27 @@ impl<'m> Thread<'m> {
     }
 
     fn exit_label(mut self) -> ThreadResult<'m> {
+        let values_cnt = self.last_frame_values_cnt();
```

Comment:
I simplified this a bit. We don't need to compute this because when we enter the `if`, we know there was exactly one label.